### ISSUE TITLE
NTR: Imaging: esophagus short (Lab: radiology)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8988]
New term request Imaging: esophagus short (elthat_101002180512): Esophageal; barium swallow, CT, MRI, or endoscopy  UMLS: 1401774

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000382
- label: Short esophagus
- parents: [Abnormal esophagus morphology (HP:0002031)](https://hpo.jax.org/app/browse/term/HP:0002031)
- pmid: PMID:29262377
- def: Abnormal reduction in the length of the esophagus.
- comment: The short esophagus was defined by surgeons as the inability to reduce the gastroesophageal (GE) junction below the diaphragm during surgery, due to inadequate esophageal mobilization. Short esophagus was also defined by endoscopists when the GE junction is positioned in adults at a distance less than 40 cm from the mouth (normal value is approximately 45 cm).
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8988
- synonyms:
